Exercise2/Plotter.py

```diff
@@ -0,0 +1,39 @@
+import matplotlib.pyplot as plt
+import math
+import numpy as np
+
+#Creates class
+class Plot(object):
+    def __init__(self):
+        self.t = np.linspace(-10,10,1000)
+        self.y = np.linspace(-10,10,1000)
+        self.i = 0
+
+    def Lambda(self, t):
+        return 5 * math.sin(2 * t * math.pi)
+
+    def h(self, t):
+        return 3*math.pi**(0-self.Lambda(t))
+
+    def execute(self):
+        for item in self.t:
+            self.y[self.i] = self.h(self.t[self.i])
+            self.i +=1
+        plt.plot(self.t, self.y)
+        plt.show()
+
+#Executes code
+plot = Plot()
+plot.execute()
+
+#This fills no purpose, but the assignment was to creae
+#a child class with specific features.
+class Child(Plot):
+    def __init__(self):
+        super(Child, self).__init__()
+
+    def print_variables(self):
+        print(self.t, self.y, self.i)
```

Comment:
It's nice that you are doing this! I have a suggestion to make it more "Pythonic":

Similar to `__init__()`, there is a method called `__str__()` which you indirectly call everytime you use `print(something)`.

```suggestion
    def __str__(self):
        return '({}, {}, {})'.format(self.t, self.y, self.i)
```

And then in your line 39, you could call it like

`print(child)`

:smile: